spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -86,12 +86,12 @@ trait CommonJobExecution {
     dao.authenticate()
     val dataset = dao.getDataset(cmd.datasetName, cmd.datasetVersion)
     val reportVersion = getReportVersion(cmd, dataset)
-    val pathCfg = getPathConfig(cmd, dataset, reportVersion)
+    val pathCfg: PathConfig = getPathConfig(cmd, dataset, reportVersion)
```

Comment:
Not a problem, just curious - why did you add the type declaration? :-)

Reply:
Good question. Might be a misclick.